CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 - Added: `block-closing-brace-empty-line-before` rule.
 - Added: `custom-property-empty-line-before` rule.
+- Added: `declaration-empty-line-before` rule.
+- Added: `media-feature-name-case` rule.
```

Comment:
This isn't supposed to be in this PR, was added in https://github.com/stylelint/stylelint/commit/c1dd30891aa061dc39382f70ff7b2454087116fc